src/http/routes.js

```diff
@@ -25,5 +25,6 @@ router.post('/users/login', controllers.allUser.login);
 
 router.get('/device', authMiddleware, controllers.device.getAllDevice);
 router.get('/device/:deviceId', authMiddleware, controllers.device.getSingleDevice);
+router.put('/device/:deviceId', authMiddleware, controllers.device.updateDevice);
```

Comment:
Only hospital and ward users should be able to update device info, please make use of the `hasUserPrivledge` here.

Reply:
fixed